daemon/algod/api/server/v2/dryrun_test.go

```diff
@@ -1634,3 +1643,126 @@ int 1`)
 		logResponse(t, &response)
 	}
 }
+
+func checkEvalDelta(t *testing.T,
+	response generated.DryrunResponse,
+	expectedGlobalDelta generated.StateDelta,
+	expectedLocalDelta generated.AccountStateDelta,
+) {
+	for _, rt := range response.Txns {
+		if rt.GlobalDelta != nil && len(*rt.GlobalDelta) > 0 {
+			assert.Equal(t, len(expectedGlobalDelta), len(*rt.GlobalDelta))
+			assert.Equal(t, *rt.GlobalDelta, expectedGlobalDelta)
+		} else {
+			assert.Nil(t, expectedGlobalDelta)
+		}
+
+		if rt.LocalDeltas != nil {
+			for _, ld := range *rt.LocalDeltas {
+				assert.Equal(t, expectedLocalDelta.Address, ld.Address)
+				assert.Equal(t, len(ld.Delta), len(expectedLocalDelta.Delta))
+				assert.Equal(t, ld.Delta, expectedLocalDelta.Delta)
```

Comment:
I probably confused the issue by mentioning length, but now that you use assert.Equal, you don't need the length check.

Reply:
thanks for catching, should be deleted in the latest commit